modules/julia/cmake/hooks/STATUS_DUMP_EXTRA.cmake

```diff
@@ -0,0 +1,13 @@
+# ========================= julia =========================
+if(WITH_JULIA OR Julia_FOUND)
+  status("")
+  status("  JULIA:" Julia_FOUND THEN "YES" ELSE "NO")
```

Comment:
use `HAVE_JULIA THEN "YES" ELSE "NO"`, because this line should indicate whether Julia bindings will be built or not.